Lib/email/contentmanager.py

```diff
@@ -3,6 +3,7 @@
 import email.message
 import email.errors
 from email import quoprimime
+import sys
```

Comment:
```suggestion
import email.errors
import sys
from email import quoprimime
```

we usually put `import [...]` before `from` imports and sort them alphabetically within those groups.

Reply:
Thanks, sorry I wasn't aware of that rule before!

Reply:
It's not a strict rule, but the rule is to stick to existing conventions. If the diff is still small if someone uses a sorted import rule, then it's fine, but we don't want people changing the imports order if this introduces a too large diff.

Reply:
Got it, I will take note of the feature.